server/src/test/java/org/elasticsearch/rest/action/document/RestBulkActionTests.java

```diff
@@ -51,7 +51,7 @@ public void testBulkPipelineUpsert() throws Exception {
         new RestBulkAction(settings(Version.CURRENT).build(), mock(RestController.class))
```

Comment:
In other PRs we have added a dedicated test here to ensure we're emitting deprecation warnings. However, there are several integration tests in this PR that maked typed calls and check for deprecation warnings, so I don't think it's strictly necessary.